scripts/rabbitmq-server-ha.ocf

```diff
@@ -1445,7 +1445,7 @@ get_monitor() {
       fi
     fi
 
-    if [ $rc -eq $OCF_ERR_GENERIC -o $rc_check -eq $OCF_ERR_GENERIC ]; then
+    if [ "$rc" -eq "$OCF_ERR_GENERIC" -o "${rc_check:-$OCF_SUCCESS}" -eq "$OCF_ERR_GENERIC" ]; then
```

Comment:
To remove warnings from script output.
As descriptions says:
- "Without double quotes it complains about misplaced '-eq' operator" - rc_check is not always initialized
- "With quotes but without default value it sometimes complains about bad integer" - empty string is still not a valid integer